MSGraphSDKNXOAuth2/Auth/NXOAuth2AuthenticationProvider.m

```diff
@@ -26,6 +26,8 @@
 #import "MSAuthenticationViewController.h"
 #import "MSLogger.h"
 
+static NSString *const lastIdentifierKey = @"lastIdentifierKey";
```

Comment:
Unique key like "com.graph.nxoauth2.lastIdentifierKey" to avoid collisions with consuming app